### PR TITLE
Gearset logo

### PR DESCRIPTION
**Pull request by CatherineBacon**
Gearset logo surrounded by orange lights

### CODE REVIEW

**Comment by CatherineBacon** [comment]
@olane let me know if you have any comments about this code.

File: model-generator.js [hunk 1]
```diff
@@ -1,11 +1,9 @@
-
-
-let numOrgs = 4;
+let numOrgsAndLogo = 4 + 1;
 let numPixelsPerOrg = 45;
 
 let model = [];
 
-for(var orgIndex = 0; orgIndex < numOrgs; orgIndex++){
+for(var orgIndex = 0; orgIndex < numOrgsAndLogo; orgIndex++){
```
Comment (olane) [inline model-generator.js]: Hm, feels a bit messy to have the extra light as an "org" - I'd probably add an extra property to each model item called "type" or something and have "org" and "logo" as the types like:

```
[
    {
        "type": "logo",
        "pixelIndex": 0
    },
    {
        "type": "logo",
        "pixelIndex": 1
    },
...
...
    {
        "type": "org",
        "orgIndex": 0,
        "pixelIndex": 0
    },
    {
        "type": "org",
        "orgIndex": 0,
        "pixelIndex": 1
    },
...
...
    {
        "type": "org",
        "orgIndex": 1,
        "pixelIndex": 0
    },
    {
        "type": "org",
        "orgIndex": 1,
        "pixelIndex": 1
    },
...
```

That way, in the `draw` method you can switch on `modelPoint.type` and have a different method called when it's the logo, and you don't have to add a logo "org state".

File: fcserver.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
 				[ 0,   0,   0, 45 ],
 				[ 0,  45,  64, 45 ],
 				[ 0,  90, 384, 45 ],
-				[ 0, 135, 448, 45 ]
+				[ 0, 135, 448, 45 ],
+				[ 0, 180, 128, 45 ]
```
Comment (olane) [inline fcserver.json]: Probably worth adding this to the README

File: stateToColourMapping.js [hunk 3]
```diff
@@ -14,6 +14,7 @@ stateColours[states.failing] = red;
 stateColours[states.unknown] = white;
 
 function getOrgColour(orgState){
+    if (orgState === "gearsetLogo") return orange;
```
Comment (olane) [inline stateToColourMapping.js]: I think if we're going to go down the road of the logo being a specialised "org" then we should just admit that and add a "logo" state to `states.js` and then we don't need this special case. My other comment suggests a way we can get around doing that, though :)

File: model-generator.js (hunk 1 above)
Comment (CatherineBacon) [reply model-generator.js]: hmmm, yeah, this didn't feel quite right when I was writing it. Adding "type" is much nicer

File: fcserver.json (hunk 2 above)
Comment (CatherineBacon) [reply fcserver.json]: updated.

File: stateToColourMapping.js (hunk 3 above)
Comment (CatherineBacon) [reply stateToColourMapping.js]: Added a "type" :-)

**Pull request merged**